src/ert/scheduler/lsf_driver.py

```diff
@@ -508,3 +509,26 @@ def _build_resource_requirement_arg(self) -> List[str]:
 
     async def finish(self) -> None:
         pass
+
+    def read_output_files(
+        self, runpath: str, job_name: str, num_tail_characters_to_read: int = 300
+    ) -> str:
+        error_msg = ""
+        stderr_file = Path(runpath) / (job_name + ".LSF-stderr")
+        if msg := get_info_from_text_outfile(stderr_file, num_tail_characters_to_read):
+            error_msg += f"\n\t LSF-err: {msg}"
+        stdout_file = Path(runpath) / (job_name + ".LSF-stdout")
+        if msg := get_info_from_text_outfile(stdout_file, num_tail_characters_to_read):
+            error_msg += f"\n\t LSF-out: {msg}"
+        return error_msg
+
+
+def get_info_from_text_outfile(file_path: Path, num_chars: int) -> str:
+    if not file_path.exists():
+        return f"No output files for {file_path}"
+    with open(file_path, encoding="utf-8") as file:
+        file.seek(0, 2)
```

Comment:
Can we change this to `file.seek(num_chars, 2)` instead? Would seek default to position 0 if trying to go before the start of the file?

Reply:
When opening in text mode this will not work as `file.seek(x, 2)` requires a byte offset and not in terms of number of characters.